src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -211,7 +213,14 @@ export default class MutationMapper<
                 trackVisibility={this.trackVisibility}
                 trackDataStatus={this.trackDataStatus}
                 onTrackVisibilityChange={this.onTrackVisibilityChange}
-                getLollipopColor={getColorForProteinImpactType}
+                getLollipopColor={(mutations: any) =>
```

Comment:
why do we need to make `mutations` any, here?

Reply:
I tried to make it more specific but, for whatever reason, theres some weird stuff going on with which `Mutation` is being used. I spent about 30 minutes trying to untangle it then just decided "if it works now then whatever, just get TS off my back"

Reply:
i think we should spend a little more time figuring it out since it's an important type.  maybe pair program?